diff --git a/node_modules/react-native-add-calendar-event/ios/AddCalendarEvent.m b/node_modules/react-native-add-calendar-event/ios/AddCalendarEvent.m
index d053473..699ca86 100644
--- a/node_modules/react-native-add-calendar-event/ios/AddCalendarEvent.m
+++ b/node_modules/react-native-add-calendar-event/ios/AddCalendarEvent.m
@@ -52,6 +52,7 @@ - (NSDictionary *)constantsToExport
 static NSString *const _notes = @"notes";
 static NSString *const _url = @"url";
 static NSString *const _allDay = @"allDay";
+static NSString *const _alert = @"alert";
 
 static NSString *const MODULE_NAME= @"AddCalendarEvent";
 
@@ -264,6 +265,33 @@ - (EKEvent *)createNewEventInstance {
     if (options[_allDay]) {
         event.allDay = [RCTConvert BOOL:options[_allDay]];
     }
+    if (options[_alert]) {
+        NSDate *originalDate =  [RCTConvert NSDate:options[_startDate]];
+
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"0"] == NSOrderedSame)
+        {
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:originalDate];
+            event.alarms = @[alarm];
+        }
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"1"] == NSOrderedSame)
+        {
+            NSDate *alertReminder = [originalDate dateByAddingTimeInterval:-60*5];
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:alertReminder];
+            event.alarms = @[alarm];
+        }
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"2"] == NSOrderedSame)
+        {
+            NSDate *alertReminder = [originalDate dateByAddingTimeInterval:-60*30];
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:alertReminder];
+            event.alarms = @[alarm];
+        }
+        if ([[RCTConvert NSString:options[_alert]] caseInsensitiveCompare:@"3"] == NSOrderedSame)
+        {
+            NSDate *alertReminder = [originalDate dateByAddingTimeInterval:-60*60];
+            EKAlarm * alarm = [EKAlarm alarmWithAbsoluteDate:alertReminder];
+            event.alarms = @[alarm];
+        }
+    }
     return event;
 }
 
